src/pages/collection/series/SeriesImages.tsx

```diff
@@ -96,7 +97,16 @@ const SeriesImages = () => {
           <Button
             buttonType="primary"
             className="rounded-md border border-panel-border px-4 py-3 font-semibold"
-            disabled
+            disabled={!Object.keys(selectedImage).length}
+            onClick={() => {
+              changeImage({
+                seriesId,
+                imageType: selectedImage.Type,
+                params: { ID: selectedImage.ID, Source: selectedImage.Source },
+              })
+                .then(() => toast.success(`Series ${selectedImage.Type} has been changed.`))
+                .catch(console.error);
+            }}
           >
             Set As Series Poster
```

Comment:
The 'Poster' part should be dynamic based on the currently selected image view imo.

Reply:
Either what revam said or just make it "Set as default"

Reply:
That also works. 👍